release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -2644,3 +2710,39 @@ This update contains changes from Kubernetes 1.23.3 up to 1.23.5. More informati
+
+[id="ocp-4-10-8"]
+=== RHSA-2022:1162 - {product-title} 4.10.8 bug fix and security update
+
+Issued: 2022-04-07
+
+{product-title} release 4.10.8, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHSA-2022:1162[RHSA-2022:1162] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2022:1161[RHBA-2022:1161] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6877401[{product-title} 4.10.8 container image list]
+
+[id="ocp-4-10-8-removed-feature-azure-mint-mode"]
+==== Removed features
+
+Starting with {product-title} 4.10.8, support for Google Cloud Platform Workload Identity has been removed from {product-title} 4.10 for the image registry. This change is due to the discovery of link:https://bugzilla.redhat.com/show_bug.cgi?id=2069807[an adverse impact to the image registry]. For more information, see xref:../release_notes/ocp-4-10-release-notes.adoc#ocp-4-10-removed-feature-auth-gcp-workload-identity[Image registry support for Google Cloud Platform Workload Identity removed].
+
+[id="ocp-4-10-8-known-issues"]
+==== Known issues
+
+* Currently, the web console does not display virtual machine templates that are deployed to a custom namespace. Only templates deployed to the default namespace are displayed in the web console. As a workaround, avoid deploying templates to a custom namespace. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2054650[*BZ#2054650*])
+
+[id="ocp-4-10-8-bug-fixes"]
+==== Bug fixes
+* Previously, the Infrastructure Operator could not provision X11 and X12 based systems due to validation errors created by the bare metal controller (BMC) when special characters such as question marks or equals signs were used in the `filename` parameters of URLs. With this update, the `filename` parameter is removed from the URL if the virtual media image is backed by a local file. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2011626[*BZ#2011626*])
+
+* Previously, when cloning a virtual machine from a template, changes the Operator made reverted after dismissing the dialog box if the boot disk was edited and the storage class was changed. With this update, changes made to storage class remain set after closing the dialogue box. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2049762[*BZ#2049762*])
+
+* Previously, the `startupProbe` field was added to container’s definition. As a result, `startupProbe` causes problems when creating a debug pod. With this update, `startupProbe` is removed by default from the debug pod by the `Expose --keep-startup flag` parameter, which is now set to false by default.(link:https://bugzilla.redhat.com/show_bug.cgi?id=2068474[*BZ#2068474*])
```

Comment:
* Should this say "to a container's definition"?
* Needs a space between the last period and the bug link parenthesis